pymc/gp/cov.py

```diff
@@ -595,6 +600,28 @@ def power_spectral_density(self, omega: TensorLike) -> TensorVariable:
         exp = pt.exp(-0.5 * pt.dot(pt.square(omega), pt.square(ls)))
         return c * pt.prod(ls) * exp
 
+    @staticmethod
+    def required_num_eigenvectors(L: float, ls: float) -> int:
+        r"""Number of eigenvectors in Hilbert space that approximate the GP well.
```

Comment:
Could we add the paper (and maybe an equation reference) to understand where to learn more about the factors (for us, the newbies?)